src/Scripting/VM/VirtualMachine.hpp

```
@@ -0,0 +1,414 @@
+#include <Scripting/Utils/Bytes.hpp>
+#include <Scripting/Utils/VariantHelpers.hpp>
+#include <Scripting/Compiler/Instruction.hpp>
+#include <Scripting/Compiler/ConstantPool.hpp>
+#include <Scripting/Compiler/GraphNodeRegistry.hpp>
+
+#include "VMException.hpp"
+#include "LogicalConcept.hpp"
+#include "ArithmeticConcept.hpp"
+
+
+#define PACK_OPS(OP1, OP2)	(static_cast<Compiler::RawOperandT>(static_cast<AsTL::BYTE>(OP1) & 0xFF) << 8) | \
```

Clang (14.0.6)

[warning] function-like macro 'PACK_OPS' used; consider a 'constexpr' template function

(cppcoreguidelines-macro-usage)